guides/moving-to-oss.md

```diff
@@ -0,0 +1,227 @@
+| External PostgreSQL support             | ✅                                                                                                                   | ✅ ([configuration flag](https://coder.com/docs/coder-oss/latest/install/configure))                                                                                  |
+| External TLS documentation              | ✅ (via [cert-manager](https://coder.com/docs/coder/latest/guides/tls-certificates))                                 | ⌛ [#3518](https://github.com/coder/coder/issues/3518)                                                                                                                |
+| **Multi region/cloud (workspaces)**     | ✅ [Workspace providers](https://coder.com/docs/coder/latest/admin/workspace-providers) support additional clusters. | ✅ [Templates](https://coder.com/docs/coder/latest/admin/templates) can provision resources in any clouds, clusters, or region                                        |
+| **Multi region/cloud (provisioning)**   | ❌                                                                                                                   | ⌛ [#44](https://github.com/coder/coder/issues/44)                                                                                                                    |
+| **Multi region/cloud (authentication)** | ✅ Authenticates to clusters via secrets stored in the database                                                      | ✅ Authenticates via Terraform provider and [provisioner](https://coder.com/docs/coder-oss/latest/architecture) environment                                           |
+| **Multi region/cloud (dashboard)**      | ✅ Multi-region [satellites](https://coder.com/docs/coder/latest/admin/satellites) for faster IDE connections.       | ⌛ [#3227](https://github.com/coder/coder/issues/3227)                                                                                                                |
+| **Multi region/cloud (tunnel/SSH)**     | ✅ [Direct connections via STUN](https://coder.com/docs/coder/latest/admin/stun)                                     | ✅ Direct connections via STUN ([configuration flag](https://coder.com/docs/coder-oss/latest/install/configure))                                                      |
+
+<small>See the Coder OSS
+[installation docs](https://coder.com/docs/coder-oss/latest/install) for more
+details. Something missing, or have feedback?
+[Let us know](https://coder.com/contact)</small>
+
+### CLI
+
+Coder OSS uses a seperate
+[command line utility](https://coder.com/docs/coder-oss/latest/install). To use
+both CLIs on the same machine, you can install the Coder OSS CLI under a
+different name (e.g. `codeross`):
+
+```sh
```

Comment:
these should be separate commands with explanations in between:

codeross is installed separately from coderv1. to install codeross run the following command.
[command]

even though coder v1 and codeross are installed on the same system, you can still run coder v1 using the following command
[command]

codeross is run using the following command
[command]